crates/bevy_ecs/src/system/commands.rs

```diff
@@ -189,17 +190,50 @@ pub struct Commands {
 }
 
 impl Commands {
+    /// Creates a new entity and calls `insert` with the it and `components`.
+    ///
+    /// Note that `components` is a bundle. If you would like to spawn an entity with a single component, consider wrapping the component in a tuple (which `DynamicBundle` is implemented for).
```

Comment:
Could be an opportunity to rename `components` to `bundle` to make it clearer. type was renamed but not variable...

Reply:
I've done the rename in just commands.rs and world.rs, but am happy to walk it back if we think that this is not the right changeset for such a rename.